docs/source/interfaces/Frequency Domain Analysis.rst

```diff
@@ -46,6 +46,8 @@ All these tabs can be dragged away from the interface into their own window.
 * :ref:`Grouping Tab <muon_grouping_tab-ref>`
 * :ref:`Phase Table Tab <muon_phase_table_tab-ref>`
 * :ref:`Transform Tab <muon_transform_tab-ref>`
+* :ref:`Fitting Tab <muon_fitting_tab-ref>`
+* :ref:`Results Table Tab <muon_results_tab-ref>`
```

Comment:
On the `muon_fitting_tab.rst` and `muon_results_tab.rst` documents add Frequency domain analysis to the "User By" section.